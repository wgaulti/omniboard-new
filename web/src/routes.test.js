import React from 'react';
import Routes from './routes';

describe('Routes', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Routes/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
