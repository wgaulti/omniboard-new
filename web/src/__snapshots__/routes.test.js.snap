// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render correctly 1`] = `
<BrowserRouter>
  <div>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/:dbPath?"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </div>
</BrowserRouter>
`;
